def f(a):

    a = int(a)
    return a


try:
    f("a")
    print("teste")
except:
    print("erro")
